modules/distr-tracing-install-jaeger-operator.adoc

```diff
@@ -5,48 +5,45 @@ This module included in the following assemblies:
 
 :_mod-docs-content-type: PROCEDURE
 [id="distr-tracing-jaeger-operator-install_{context}"]
-= Installing the {JaegerName} Operator
+= Installing the {JaegerOperator} Operator
 
-To install {JaegerName}, you use the link:https://operatorhub.io/[OperatorHub] to install the {JaegerName} Operator.
+You can install the {JaegerOperator} Operator through the link:https://operatorhub.io/[*OperatorHub*].
```

Comment:
I won't block merge on this, but I would recommend making a follow-up PR to remove this `link:` here. OperatorHub.io is the upstream site for community Operators. Just saying "OperatorHub" (with no link) here is fine though / good enough, as that is what the related web console page on an actual OCP cluster is called too.

Reply:
OK. Thank you. Will do.